### DISCUSSION TITLE
HumanitZ

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#553] [Game request]
Hello, today I want to add this suggestion. 

https://store.steampowered.com/app/1766060/HumanitZ/

Is similar to Project Zomboid and a splash of DayZ. 

In their recent update, they said they're in developing dedicated servers. 

https://store.steampowered.com/news/app/1766060/view/3805038873995353792?l=english

Maybe this game can then be considered

**Comment by nghialele** [comment]
+1 for this game
